packages/pluggableWidgets/datagrid-web/src/Datagrid.xml

```diff
@@ -77,6 +54,39 @@
                                 <caption>Dynamic text</caption>
                                 <description/>
                             </property>
+                            <property key="header" type="textTemplate" required="false">
+                                <caption>Caption</caption>
+                                <description/>
+                            </property>
+                        </propertyGroup>
+                        <propertyGroup caption="Column capabilities">
+                            <property key="sortable" type="boolean" defaultValue="true">
+                                <caption>Can sort</caption>
+                                <description/>
+                            </property>
+                            <property key="resizable" type="boolean" defaultValue="true">
+                                <caption>Can resize</caption>
+                                <description/>
+                            </property>
+                            <property key="draggable" type="boolean" defaultValue="true">
+                                <caption>Can reorder</caption>
+                                <description/>
+                            </property>
+                            <property key="hidable" type="enumeration" defaultValue="yes">
+                                <caption>Can hide</caption>
```

Comment:
I feel like the word "can" makes it sound like a boolean, cuz intuitively it would be "can hide" or "can not hide". But now we're also including a default state to it. Maybe something like "Hide Settings" or "Visibility Settings" would be more appropriate? With 3 values: `Always shown`, `Can be hidden, hidden by default`, and `Can be hidden, shown by default`.

If this doesn't fit in the designs, I personally think it would be better to add a conditional property: `Can hide`: yes/no. If yes -> `Shown by default`: yes/no.

Reply:
This decision was made with 2 PMs, I am not the one deciding captions here.